docs/resources/scanner.md

```diff
@@ -3,17 +3,17 @@ layout: "wallarm"
 page_title: "Wallarm: wallarm_scanner"
 subcategory: "Common"
 description: |-
-  Provides the resource to manage Wallarm Scanner scope.
+  Provides the resource to manage Wallarm scanner scope.
```

Comment:
Scanner is capitalized

Reply:
Basing on [this](https://docs.wallarm.com/user-guides/scanner/intro/#network-scope-scanning) doc, it is not.